emulator-10.xml

```diff
@@ -34,6 +34,7 @@
   </project>
   <project groups="kai" name="platform_system_connectivity_wificond" path="system/connectivity/wificond" remote="kaios" revision="gonk_android-10.0.0_r11"/>
   <project groups="kai" name="platform_system_core" path="system/core" remote="kaios" revision="gonk_android-10.0.0_r11"/>
+  <project groups="kai" name="device_generic_goldfish" path="device/generic/goldfish" remote="kaios" revision="gonk_android-10.0.0_r11" />
```

Comment:
@lissyx, I have the branch ready, forked from aosp.  Thanks! 

https://github.com/kaiostech/device_generic_goldfish/tree/gonk_android-10.0.0_r11